### PR TITLE
[ES] Update `AGTS 2019`

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Tournaments/GTS/AGTS_2019/es.md [hunk 1]
```diff
@@ -279,209 +278,227 @@ Esta competencia ha llegado a su fin y los acreedores al podio son los siguiente
...
-   - El cambio de jugadores está permitido sin limitaciones durante toda la partida.
-   - Mismas reglas de desconexión que en las reglas normales aplican.
+1. Un árbitro creará una sala privada 10 minutos antes del partido. Los capitanes de ambos equipos serán invitados a la sala cuando sea la hora del partido. A continuación, los árbitros o los capitanes invitarán a los demás jugadores a entrar en la sala.
+2. Si no hay personal o árbitro disponible, el partido se aplazará.
+3. Si asisten menos jugadores del mínimo requerido, el tiempo máximo que se puede aplazar el partido es de 10 minutos.
+4. Si ninguno de los dos equipos tiene el mínimo de jugadores requeridos después de 10 minutos, contará como una derrota para ambos equipos en la fase de grupos. En el cuadro de doble eliminación, avanzará el equipo mejor clasificado.
+5. El partido se jugará como TeamVs y ScoreV2 y el nombre de la sala debe ser:  «AGTS 2019: (Equipo Rojo) vs (Equipo Azul)».
+6. A cada equipo se le permite un calentamiento, pero debe ser inferior a 4 minutos. Todo el mundo está permitido en la sala durante los calentamientos.
```
Comment (UnknownOsu) [inline wiki/Tournaments/GTS/AGTS_2019/es.md:477]:
```suggestion
6. A cada equipo se le permite un calentamiento, pero debe ser inferior a 4 minutos. Todo el mundo puede entrar en la sala durante el calentamiento.
```

File: wiki/Tournaments/GTS/AGTS_2019/es.md [hunk 2]
```diff
@@ -279,209 +278,227 @@ Esta competencia ha llegado a su fin y los acreedores al podio son los siguiente
...
+7. Cada capitán tiene que prohibir **un beatmap** para ser seleccionado del grupo (aparte del desempate). Estos beatmaps no podrán ser elegidos por ningún equipo en todo el partido.
```
Comment (UnknownOsu) [inline wiki/Tournaments/GTS/AGTS_2019/es.md:478]:
```suggestion
7. Cada capitán tiene que banear **un beatmap** para ser seleccionado del mappool (aparte del desempate). Estos beatmaps no podrán ser elegidos por ningún equipo en todo el partido.
```

File: wiki/Tournaments/GTS/AGTS_2019/es.md [hunk 3]
```diff
@@ -279,209 +278,227 @@ Esta competencia ha llegado a su fin y los acreedores al podio son los siguiente
...
+8. El capitán de cada equipo tirará con el comando `!roll` en el chat.
+9. El equipo con **la tirada más alta prohibe al segundo y elige primero**. El equipo con **la tirada más baja prohibe al primero y elige segundo**.
```
Comment (UnknownOsu) [inline wiki/Tournaments/GTS/AGTS_2019/es.md:480]:
```suggestion
9. El equipo con **la tirada más alta banea segundo y elige primero**. El equipo con **la tirada más baja banea primero y elige segundo**.
```

File: wiki/Tournaments/GTS/AGTS_2019/es.md [hunk 4]
```diff
@@ -279,209 +278,227 @@ Esta competencia ha llegado a su fin y los acreedores al podio son los siguiente
...
+10. La selección del beatmap alternará entonces entre cada capitán seleccionando un beatmap de la piscina de mapas.
```
Comment (UnknownOsu) [inline wiki/Tournaments/GTS/AGTS_2019/es.md:481]:
```suggestion
10. La selección del beatmap alternará entonces entre cada capitán seleccionando un beatmap del mappool.
```

File: wiki/Tournaments/GTS/AGTS_2019/es.md [hunk 5]
```diff
@@ -279,209 +278,227 @@ Esta competencia ha llegado a su fin y los acreedores al podio son los siguiente
...
+18. En caso de empate a puntos (ejemplo: 4-4 en BO9), se elige el desempate.
+
+### Reglas del all-star
+
+All-star es un divertido evento que tendrá lugar el mismo fin de semana que la gran final. En él se enfrentarán los mejores jugadores del torneo en dos tipos de partidos diferentes.
+
+1. En este evento participarán 12 jugadores. Se decidirán por votación.
+2. Los 12 mejores jugadores jugarán en el all-star. Si hay más de 2 jugadores de un país entre los 12 más votados solo los 2 mejores participarán en el all-star.
+3. Los 2 jugadores más votados serán los capitanes de cada equipo y elegirán en la lista de los siguientes 10 jugadores más votados. El jugador más votado empezará a elegir primero. La votación se hará al estilo «snake draft picking», así es como funcionará: A->B->B->A->A->B->B->A->A->B.
+4. En primer lugar, se celebrarán 2 combates de battle royale 3 contra 3 con un mappool de 12 mapas (2 NM,HR,HD,DT,FM,EX). (EX = mapas más difíciles) Los mapas se eligen al azar con un generador de nombres aleatorios a través de la rueda giratoria.
```
Comment (UnknownOsu) [inline wiki/Tournaments/GTS/AGTS_2019/es.md:498]:
```suggestion
4. En primer lugar, se llevarán a cabo 2 partidos de battle royale 3 contra 3 con un mappool de 12 mapas (2 NM,HR,HD,DT,FM,EX). (EX = mapas más difíciles) Los mapas se eligen al azar con un generador de nombres aleatorios a través de la rueda giratoria.
```

File: wiki/Tournaments/GTS/AGTS_2019/es.md [hunk 6]
```diff
@@ -279,209 +278,227 @@ Esta competencia ha llegado a su fin y los acreedores al podio son los siguiente
...
+5. También se celebrará un partido 4 contra 4 con las siguientes reglas:
```
Comment (UnknownOsu) [inline wiki/Tournaments/GTS/AGTS_2019/es.md:499]:
```suggestion
5. También se realizará un partido 4 contra 4 con las siguientes reglas:
```

File: wiki/Tournaments/GTS/AGTS_2019/es.md [hunk 7]
```diff
@@ -279,209 +278,227 @@ Esta competencia ha llegado a su fin y los acreedores al podio son los siguiente
...
+   - Mappool de 5 NM, HR, HD, DT, FM, EX, y 1 TB (mismas reglas para mods que los partidos normales).
```
Comment (UnknownOsu) [inline wiki/Tournaments/GTS/AGTS_2019/es.md:500]:
```suggestion
   - Mappool de 5 NM, HR, HD, DT, FM, EX, y 1 TB (mismas reglas para los mods que para los partidos normales).
```

File: wiki/Tournaments/GTS/AGTS_2019/es.md [hunk 8]
```diff
@@ -279,209 +278,227 @@ Esta competencia ha llegado a su fin y los acreedores al podio son los siguiente
...
+   - No se dispondrá de canalizaciones de mapas.
+   - El mod utilizado se elegirá al azar según una tirada (de acuerdo con una tabla) y después se elegirá el mapa.
+   - El intercambio de jugadores durante una partida está permitido sin limitaciones.
```
Comment (UnknownOsu) [inline wiki/Tournaments/GTS/AGTS_2019/es.md:503]:
```suggestion
   - El intercambio de jugadores durante un partido está permitido sin limitaciones.
```

File: wiki/Tournaments/GTS/AGTS_2019/es.md [hunk 9]
```diff
@@ -279,209 +278,227 @@ Esta competencia ha llegado a su fin y los acreedores al podio son los siguiente
...
+   - Mismas reglas que en una partida normal para las desconexiones.
```
Comment (UnknownOsu) [inline wiki/Tournaments/GTS/AGTS_2019/es.md:504]:
```suggestion
   - Mismas reglas que en un partido normal para las desconexiones.
```

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/GTS/AGTS_2019/es.md [hunk 10]
```diff
@@ -3,124 +3,123 @@ tags:
...
 
 ## Premios
 
-| Puestos | Premio(s) |
+| Posición | Premios |
 | :-: | :-- |
-| ![Corona de Oro](/wiki/shared/crown-gold.png "1er lugar") | Medalla única de perfil, banner de equipo exclusivo |
-| ![Corona de Plata](/wiki/shared/crown-silver.png "2ndo lugar") | Banner de equipo exclusivo |
-| ![Corona de Bronce](/wiki/shared/crown-bronze.png "3er lugar") | Banner de equipo exclusivo |
+| ![Corona de oro](/wiki/shared/crown-gold.png "Primer lugar") | Insignia de perfil única, pancarta exclusiva del equipo |
```
[the remark this answers is not in the document]
Comment (agatemosu) [reply wiki/Tournaments/GTS/AGTS_2019/es.md:31]: AJAKAJS si

**Review by TicClick — APPROVED** [review]

**Pull request merged**